### DISCUSSION TITLE
Users can not create items

### DISCUSSION THREAD

**Discussion by bcsrgf** [nilsteampassnet/TeamPass#4287] [Q&A]
LDAP users or local users can not create items.

Current version loaded 3.1.2.45

Ubuntu 24.04 LTS

Server is connected to internet

PHP version: 8.3.6
Memory limit: 128M
Memory usage: 525.62 KB
Maximum time execution: 30
Maximum file size upload: 2M

Server version: 9.0.1
Database size: 0.94MB

![image](https://github.com/user-attachments/assets/3feb8bf7-5067-483a-af1e-02fa9df863fe)
Task user_build_cache never end

When users try to create a item, even at personal folder, receive a message "You are not allowed to do that!"

**Comment by corentin-soriano** [comment]
Are you in sequential tree load strategy mode?
If so, can you try it in full mode?

**Comment by bcsrgf** [comment]
Thanks for your reply, we already using a full load strategy
